assets/src/main/resources/META-INF/services/bisq.asset.Asset

```diff
@@ -4,11 +4,14 @@
 # See https://bisq.network/list-asset for complete instructions.
 bisq.asset.coins.Actinium
 bisq.asset.coins.Aeon
-bisq.asset.coins.BitcoinClashic
-bisq.asset.coins.BitcoinRhodium
+bisq.asset.coins.Australiacash
```

Comment:
Please rebase on top of master and make sure that your commit adds only a single line into this file.

Reply:
I give up.  This is a constantly moving target since you modify that file at will causing conflicts well after I post my pull request.  Been trying to satisfy your requirements since November.  Good luck.